cmd/dagger/cmd/root.go

```diff
@@ -21,6 +23,16 @@ var rootCmd = &cobra.Command{
 }
 
 func init() {
+	// filter out useless commandConn.CloseWrite warning message that can occur
+	// when dagger runs for the first time. This should be fixed upstream:
+	// unreachable: "commandConn.CloseWrite: commandconn: failed to wait: signal: killed"
+	// https://github.com/docker/cli/blob/3fb4fb83dfb5db0c0753a8316f21aea54dab32c5/cli/connhelper/commandconn/commandconn.go#L203-L214
+	logrus.AddHook(logutil.NewFilter([]logrus.Level{
```

Comment:
Can't we achieve the same using zerolog hooks? https://github.com/rs/zerolog#hooks

So we don't pull another dependency

Reply:
I would have liked to but upstream it using logrus unfortunately: https://github.com/docker/cli/blob/3fb4fb83dfb5db0c0753a8316f21aea54dab32c5/cli/connhelper/commandconn/commandconn.go#L203-L214 so don't think we can workaround that.

Reply:
Also it's a transient dep so currently used anyway I think: https://github.com/dagger/dagger/blob/464c7ad708ba93e15df08d71ab3f552539b243f6/go.sum#L1313-L1314

Reply:
👍